test/testdata/rewriter/t_enum_override_initialize.rb

```diff
@@ -0,0 +1,14 @@
+# typed: strict
+
+class MyEnum < T::Enum
+  extend T::Sig
+
+  sig {params(x: String, y: String).void}
+  def initialize(x, y); end
```

Comment:
Since this is a pattern we don't really want to encourage (subclassing `T::Enum`), especially not without calling `super`, I wonder if this should call `super()` just for show.

Reply:
sure, but it doesn't change the test

Reply:
calls to `super` are not typechecked

Reply:
Ack. I just meant so someone doesn't think this is code they should try elsewhere. It would be a noop purely "for show."